test/integration_tests/test_end_to_end.py

```diff
@@ -252,7 +246,58 @@ def test_our_decryption_key():
         decryption_key_path=Path(__file__).parents[2].joinpath('dummy_keys',
                                                                'our_dummy_private.asc'),
         decryption_key_passphrase='test',
-        expected='test_uac|test_caseref|Mr|Test|McTest|123 Fake Street|Duffryn||Newport|NPXXXX|P_IC_ICL1\n')
+        expected='0|test_caseref|Mr|Test|McTest|123 Fake Street|Duffryn||Newport|NPXXXX|P_IC_ICL1\n')
+
+
+def test_icl1e_with_duplicate_uacs_is_quarantined():
```

Comment:
Might be worth adding a sanity check in one / both of these to make sure a file hasn't been created elsewhere other than the quarantine.

Reply:
I was wondering about that, but I was unsure how long I should leave the test polling for files before accepting that as evidence they'll never be there.

Reply:
I was more suggesting that we should be checking that files are never in two places but perhaps that's over zealous.